### ISSUE TITLE
Using special characters to start text

### ISSUE THREAD

**Issue by Omiminpo** [inkle/ink#275]
This might be a very basic question, but I couldn't find an answer in the documentation.

Is there a way to start a sentence with characters like "*", without them being read as choices?

**Comment by joethephish** [comment]
Yup, you can use the backslash character to escape the character that follows, so: `\*ring*`.

**Comment by Omiminpo** [comment]
Thanks for the answer! :D
